src/DssSpell.sol

```diff
@@ -50,90 +43,68 @@ contract DssSpellAction is DssAction {
-    address internal constant SPARK_D3M_POOL            = 0x8b6Ae79852bcae012CBc2244e4ef85c61BAeCE35;
-    address internal constant SPARK_D3M_ORACLE          = 0xa07C4eDB18E4B3cfB9B94D9CD348BbF6d5a7f4c2;
+    uint256 internal constant WAD                            = 10 ** 18;
+    uint256 internal constant MILLION                        = 10 ** 6;
 
-    address internal constant SPARK_ADAI                = 0x4480b29AB7a1b0761e6d0d480325de28B7266d73;
-    address internal constant SPARK_DAI_STABLE_DEBT     = 0xD72630D78157E1a2feD7A329873Bfd496704403D;
-    address internal constant SPARK_DAI_VARIABLE_DEBT   = 0xa99d874d26BdfD94d474Aa04f4f7861DCD55Cbf4;
-    address internal constant SPARK_POOL_CONFIGURATOR   = 0xe0C7ec61cC47e7c02b9B24F03f75C7BC406CCA98;
+    address internal immutable STARKNET_DAI_BRIDGE           = DssExecLib.getChangelogAddress("STARKNET_DAI_BRIDGE");
 
-    address internal constant WBTC                      = 0x91277b74a9d1Cc30fA0ff4927C287fe55E307D78;  // Please note this is not the same WBTC as in Maker
-    address internal constant INTEREST_RATE_STRATEGY    = 0x491acea4126E48e9A354b64869AE16b2f27BE333;
+    function actions() public override {
+        // ---------- Starknet ----------
+        // Increase L1 Starknet Bridge Limit from 1,000,000 DAI to 5,000,000 DAI
+        // Forum: https://forum.makerdao.com/t/april-26th-2023-spell-starknet-bridge-limit/20589
+        StarknetLike(STARKNET_DAI_BRIDGE).setCeiling(5 * MILLION * WAD);
 
-    uint256 internal constant MILLION = 10 ** 6;
-    uint256 internal constant WAD = 10 ** 18;
-    uint256 internal constant RAD = 10 ** 45;
+        // ---------- Risk Parameters Changes (Stability Fee & DC-IAM) ----------
+        // Poll: https://vote.makerdao.com/polling/QmYFfRuR#poll-detail
+        // Forum: https://forum.makerdao.com/t/out-of-scope-proposed-risk-parameters-changes-stability-fee-dc-iam/20564
 
-    function actions() public override {
+        // Increase ETH-A Stability Fee by 0.25% from 1.5% to 1.75%.
+        DssExecLib.setIlkStabilityFee("ETH-A", ONE_PT_SEVEN_FIVE_PCT_RATE, true);
+
+        // Increase ETH-B Stability Fee by 0.25% from 3% to 3.25%.
+        DssExecLib.setIlkStabilityFee("ETH-B", THREE_PT_TWO_FIVE_PCT_RATE, true);
+
+        // Increase ETH-C Stability Fee by 0.25% from 0.75% to 1%.
+        DssExecLib.setIlkStabilityFee("ETH-C", ONE_PCT_RATE, true);
+
+        // Increase WSTETH-A Stability Fee by 0.25% from 1.5% to 1.75%.
+        DssExecLib.setIlkStabilityFee("WSTETH-A", ONE_PT_SEVEN_FIVE_PCT_RATE, true);
+
+        // Increase WSTETH-B Stability Fee by 0.25% from 0.75% to 1%.
+        DssExecLib.setIlkStabilityFee("WSTETH-B", ONE_PCT_RATE, true);
+
+        // Increase RETH-A Stability Fee by 0.25% from 0.5% to 0.75%.
+        DssExecLib.setIlkStabilityFee("RETH-A", ZERO_PT_SEVEN_FIVE_PCT_RATE, true);
+
+        // Increase CRVV1ETHSTETH-A Stability Fee by 0.25% from 1.5% to 1.75%.
+        // NOTE: ignore in goerli
```

Comment:
Why?

Reply:
Ilk isn't on Goerli

Reply:
The reason should be in the comment for people looking at this in the future, as it's not obvious.